services/node_create.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.consensus">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+import "basic_types.proto";
+
+/**
+ * See [NodeService.createNode()](#proto.NodeService)
+ */
+message NodeCreateTransactionBody {
+    /**
+     * node account id, mandatory field
```

Comment:
Very good questions! I don't know the answers. I asked these in our dynamic-addressbook channel.

Reply:
Added: node account id, mandatory field, ALIAS is not allowed, only ACCOUNT_NUM
     If account_id does not exist, it will reject the transaction.

Also added two Keys fields, operator_key and council_key.